modules/repository-azure/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -12,6 +12,8 @@ grant {
   permission java.net.SocketPermission "*", "connect";
   // io.netty.util.concurrent.GlobalEventExecutor.startThread
   permission java.lang.RuntimePermission "setContextClassLoader";
+  // io.netty.util.concurrent.GlobalEventExecutor.startThread
+  permission java.lang.RuntimePermission "getClassLoader";
```

Comment:
This needed because this code was added in netty: https://github.com/netty/netty/blob/4.1/common/src/main/java/io/netty/util/concurrent/GlobalEventExecutor.java#L242